### PR TITLE
PEP8 fix

### PR DESCRIPTION
**Pull request by vkmrishad**
### Checklist
- [x] I have made a material change to the repo (functionality, testing, spelling, grammar)
- [x] I have read the [Contribution Guide] and my PR follows them.
- [x] I updated my branch with the master branch.
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation about the functionality in the appropriate .md file
- [ ] I have added inline documentation to the code I modified

### Short description of what this PR does:
-  PEP8 fix

If you have questions, please send an email to [Sendgrid](mailto:dx@sendgrid.com), or file a Github Issue in this repository.

### CODE REVIEW

**Comment by SendGridDX** [comment]
[CLA assistant check](https://cla.sendgrid.com/sendgrid/smtpapi-python?pullRequest=87) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.sendgrid.com/sendgrid/smtpapi-python?pullRequest=87) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.sendgrid.com/check/sendgrid/smtpapi-python?pullRequest=87) it.</sub>

**Comment by vkmrishad** [comment]
@misterdorm

File: examples/example.py [hunk 1]
```diff
@@ -3,33 +3,33 @@
 
 import time
 
+from os import sys, path
+from smtpapi import SMTPAPIHeader
+
 if __name__ == '__main__' and __package__ is None:
-    from os import sys, path
     sys.path.append(path.dirname(path.dirname(path.abspath(__file__))))
```
Comment (misterdorm) [inline examples/example.py]: Was this a change for PEP8?  I think we may need to make sure the sys.path.append comes before importing smtpapi.

Comment (vkmrishad) [reply examples/example.py]: @misterdorm 
Not completely related to PEP8. Just redefined import order.
But now it showing conflict. I will fix the issue asap.

Comment (vkmrishad) [reply examples/example.py]: Fixed.
Replace file with Master Branch code.

**Comment by vkmrishad** [comment]
@misterdorm
Please add available labels and merge this PR

**Comment by thinkingserious** [comment]
Hello @vkmrishad,<br /><br/>Thanks again for the PR!<br />                                              <br />It's <a href="https://sendgrid.com/blog/hacktoberfest-2018-has-arrived/">HACKTOBERFEST</a>! We want to show our appreciation by sending                                              you some special Hacktoberfest swag. If you have not already, could you please fill out                                              <a href="https://dx.sendgrid.com/hacktoberfest/join">this form</a> so we can send it to you? Thanks!                                             <br /><br />Team SendGrid DX

**Comment by vkmrishad** [comment]
@thinkingserious @misterdorm 
Thank you

**Pull request merged**